meson.build

```diff
@@ -48,6 +48,15 @@ if use_audit
         dep_libcapng = dependency('libcap-ng', version: '>=0.6')
 endif
 
+#
+# Config: apparmor
+#
+
+use_apparmor = get_option('apparmor')
+if use_apparmor
+        dep_libapparmor = dependency('libapparmor', version: '>=3.0')
+endif
+
```

Comment:
I moved this above the `audit`-check, since `apparmor` is before `audit` in alphabetical order ;)